extension/src/extension.ts

```diff
@@ -280,22 +280,28 @@ class Extension extends Disposable {
     await Promise.all([
       this.repositories.create(this.getRoots()),
       this.repositoriesTree.initialize(this.getRoots()),
-      this.experiments.create(this.getRoots(), this.resourceLocator),
       this.pipelines.create(this.getRoots())
     ])
+    this.experiments.create(
```

Comment:
[F] Now that an `Experiments` needs a `Pipeline` we have to move it out of the initial promise list.